lightwood/data/timeseries_analyzer.py

```diff
@@ -28,12 +28,13 @@ def timeseries_analyzer(data: pd.DataFrame, dtype_dict: Dict[str, str],
     
     :return: Dictionary with the aforementioned insights and the `TimeseriesSettings` object for future references.
     """  # noqa
+    tss = timeseries_settings
     info = {
         'original_type': dtype_dict[target],
         'data': data[target].values
     }
-    if timeseries_settings.group_by is not None:
-        info['group_info'] = {gcol: data[gcol].tolist() for gcol in timeseries_settings.group_by}  # group col values
+    if tss.group_by is not None:
+        info['group_info'] = {gcol: data[gcol] for gcol in tss.group_by}  # group col values
```

Comment:
is pd.Series a permissible format?

Reply:
It is, all methods downstream access it with methods from `Iterable`, so all's good.